keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py

```diff
@@ -492,6 +495,7 @@ def presets_with_weights(cls):
         return {}
 
 
+@keras.utils.register_keras_serializable(package="keras_cv.models")
```

Comment:
IIRC we don't need these for the subclasses of ResNetV2Backbone, right?

Reply:
Good catch thanks!